Main.py

```diff
@@ -1,82 +1,102 @@
 import numpy as np
-#from numpy import *
-#execfile("LoadData.py")
-#execfile("InitilizeParameters.py")
-#execfile("CreateIndividuals.py")
+# from numpy import *
+# execfile("LoadData.py")
+# execfile("InitilizeParameters.py")
+# execfile("CreateIndividuals.py")
 
 from LoadData import *
 from InitilizeParameters import *
 from CreateIndividuals import simEnv
 
 # \todo: HERE, Delete variables we don't need anymore.
-#data1 = data2 = None # delete unneccesary variables.
+# data1 = data2 = None # delete unneccesary variables.
+
 
 def run():
     nr_alive = FOUNDER_COUNT
-    nr_mutations = 0
+    # nr_mutations = 0
     for iEnv in xrange(NR_ENVIRONMENTS):
         for iCycle in xrange(NR_CYCLES):
```

Comment:
Naming principles for variables in python / standard is using only lower-case with underscore. It's fine with upper-case for the variables that really are static, but `iCycle` and the like break the standard and you have vary the way you do it too, so please consider using something like `i_cycle` or `cycle_index` to be precise.